cdserver/app/models/classifiers/classifier.rb

```diff
@@ -0,0 +1,8 @@
+#
+# A Classifier is a tag or label used to group objects.
+#
+class Classifier < ApplicationRecord
```

Comment:
I almost forgot!  Classifier needs a description.  We need to copy in descriptions for the program types and shelter types and case types.  Many of those are in the glossary.  For example Hotel, Tier II all those are in the glossary.